### PR TITLE
STOR-1713: Use csi-operator for building azure-disk-operator

### PR DESCRIPTION
**Pull request by gnufied**

### CODE REVIEW

**Comment by mpatlasov** [comment]
/hold until https://github.com/openshift/csi-operator/pull/73 merged

(This PR depends on `Dockerfile.test` which is introduced by PR #73)

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-csi-operator-release-4.16-e2e-azure-csi

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-csi-operator-release-4.16-e2e-azure-csi

File: ci-operator/step-registry/storage/conf/csi-azure-disk/storage-conf-csi-azure-disk-commands.sh [hunk 1]
```diff
@@ -3,8 +3,15 @@ set -o errexit
...
 
-cd /go/src/github.com/openshift/azure-disk-csi-driver-operator
-cp test/e2e/manifest.yaml ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
+if [ -d /go/src/github.com/openshift/csi-operator/legacy/ ]; then
+    echo "Using legacy csi directory"
+    cd /go/src/github.com/openshift/csi-operator/legacy/azure-disk-csi-driver-operator
+    cp test/e2e/manifest.yaml ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
+else
+    echo "Using regular csi directory"
+    cd /go/src/github.com/openshift/aws-ebs-csi-driver-operator
```
Comment (jsafrane) [inline ci-operator/step-registry/storage/conf/csi-azure-disk/storage-conf-csi-azure-disk-commands.sh:12]: `azure-disk-csi-driver-operator`

Comment (gnufied) [reply ci-operator/step-registry/storage/conf/csi-azure-disk/storage-conf-csi-azure-disk-commands.sh]: fixed

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-azure-disk-csi-driver-release-4.15-e2e-azure-csi

**Pull request merged**